src/qt/psbtoperationsdialog.cpp

```diff
@@ -217,9 +234,17 @@ QString PSBTOperationsDialog::renderTransaction(const PartiallySignedTransaction
     }
 
     size_t num_unsigned = CountPSBTUnsignedInputs(psbtx);
+    const size_t num_partial_sigs{psbtx.inputs.at(0).partial_sigs.size()};
+    const bool all_inputs_n_partial_sigs_equal{std::all_of(psbtx.inputs.begin(), psbtx.inputs.end(),
+        [&](const PSBTInput& input) { return num_partial_sigs == input.partial_sigs.size(); })};
+
     if (num_unsigned > 0) {
         tx_description.append("<br><br>");
         tx_description.append(tr("Transaction has %1 unsigned inputs.").arg(QString::number(num_unsigned)));
+        if (all_inputs_n_partial_sigs_equal && num_partial_sigs > 0) {
+            tx_description.append(" ");
+            tx_description.append(tr("Each input has %1 partial signatures.").arg(QString::number(num_partial_sigs)));
```

Comment:
optional nit (IMO makes it more clear that includes the just added partial signature):
```suggestion
            tx_description.append(tr("Each input now has %1 partial signatures.").arg(QString::number(num_partial_sigs)));
```

Reply:
Before you sign, if there are already partial sigs on a psbt, this text will show that. For example, 3rd signer in a multisig:
- before signing `tx_description` tells them there are already 2 partial sigs
- they sign and see they have successfully added a partial signature. `tx_description` now tells them there are 3 partial sigs
- if they were to try to sign again, then they'd see the warning "Could not sign any more inputs." at the top of dialog, but `tx_description` would still show there are 3 psbts

Therefore I think `now` should not be added